### PR TITLE
chore: update fastify deps

### PR DESCRIPTION
**Pull request by alexfriesen**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/nestjs/nest/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## What is the current behavior?

Fastify moved a lot of packages around and left the remaining ones with this deprecation message: `[FST_MODULE_DEP_FASTIFY-STATIC] FastifyWarning.fastify-static: fastify-static has been deprecated. Use @fastify/static@5.0.0 instead.`

Issue Number: N/A

## What is the new behavior?

No deprecation warning.

## Does this PR introduce a breaking change?
- [x] Yes
- [ ] No

## Other information

### CODE REVIEW

**Review by micalevisk — CHANGES_REQUESTED** [review]
I guess we need to touch the `package-lock.json` file as well, right?

**Comment by alexfriesen** [comment]
> I guess we need to touch the `package-lock.json` file as well, right?

Yes you are right sorry about that. The package-lock files are updated as well

**Review by micalevisk — APPROVED** [review]

**Review by Tony133 — APPROVED** [review]

File: packages/platform-fastify/package.json [hunk 1]
```diff
@@ -17,9 +17,9 @@
     "access": "public"
   },
   "dependencies": {
-    "fastify": "3.28.0",
-    "fastify-cors": "6.0.3",
-    "fastify-formbody": "5.2.0",
+    "fastify": "3.29.0",
```
Comment (flamewow) [inline packages/platform-fastify/package.json:20]: can we add peerDependencies (I know it was not there, but we can add it as part of the PR)

```
  "peerDependencies": {
...
    "@fastify/static": "^5.0.0",
    "point-of-view": "^5.0.0"
  }
```

**Comment by flamewow** [comment]
can you check platform-fastify/interfaces/external/* and sync if needed? I believe fastifyStaticOptions got updated

File: packages/platform-fastify/package.json [hunk 2]
```diff
@@ -17,16 +17,18 @@
...
     "middie": "6.0.0",
     "path-to-regexp": "3.2.0",
     "tslib": "2.3.1"
   },
   "peerDependencies": {
     "@nestjs/common": "^8.0.0",
-    "@nestjs/core": "^8.0.0"
+    "@nestjs/core": "^8.0.0",
+    "@fastify/static": "^5.0.0",
+    "point-of-view": "^5.0.0"
```
Comment (kamilmysliwiec) [inline packages/platform-fastify/package.json:32]: we should also add the `peerDependenciesMeta` entry, flagging these deps as _optional_, see example here https://github.com/nestjs/graphql/blob/master/packages/apollo/package.json#L52

**Review by flamewow — APPROVED** [review]

**Review by Tony133 — APPROVED** [review]

**Pull request merged**